README.md

```diff
@@ -12,6 +12,7 @@ pull your dependencies back down to unbroken versions:
 ```
 cargo update --package 'serde_json' --precise '1.0.39'
 cargo update --package 'serde_derive' --precise '1.0.98'
```

Comment:
I suppose you wanted to remove this line as well..

Reply:
Oh, I think we do need it but we need to first pin `serde`.

Reply:
Did you check this?

Reply:
Yep, checked, I was right about needing to pin `serde`. Fixed.